Anima/math-tools/multi_compartment_base/animaMCMConstants.h

```diff
@@ -28,4 +29,58 @@ const double DiffusionSmallDelta = 10.0e-3;
 //! Default big delta value (classical values)
 const double DiffusionBigDelta = 40.0e-3;
 
+//! Default zero lower bound (in case we want something else than zero one day)
+const double MCMZeroLowerBound = 0.0;
+
+//! Epsilon value in case we do not want to have parameters reaching their true bounds (used for now in DDI)
+const double MCMEpsilon = 1.0e-2;
+
+//! Axial diffusivity add on to lower bound (used to ensure a minimal anisotropy to the anisotropic compartments)
+const double MCMAxialDiffusivityAddonLowerBound = 5.0e-4;
+
+//! Diffusivity lower bound for estimation
+const double MCMDiffusivityLowerBound = 1.0e-5;
```

Comment:
Some diffusivities might reach values of that order. I would use 1.0e-6 to be sure.

Reply:
Ok for that, as long as we don't go too low